assets/Css/ComponentsStyles.css

```diff
@@ -426,4 +426,56 @@ main {
 }
 .input--error-message{
   color: red;
+}
+
+/* Modal */
+
+.modal__wrapper{
+  position:relative;
+  display: flex;
+  justify-content: center;
+}
+.modal--primary-action{
+  box-shadow: 0px 0px 5px grey;
+  width:25rem;
+  padding:1rem;
+  margin:1rem;
+  border-radius: 0.5rem;
+  display: flex;
+  justify-content: center;
+}
+.modal__btn{
+  background-color: blueviolet;
+  color: white;
+  border:none;
+  padding:0.5rem 1.5rem;
+  font-weight: 800;
+  border-radius: 0.5rem;
+}
+.modal__btn--close{
+  margin-top: 8px;
+  background-color: red;
+}
+.modal--secondary-action{
+  visibility: hidden;
+  max-width:30rem;
+  min-width:15rem;
+  padding:1rem;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 1;
+  position: fixed;
+  top:50%;
+  left:50%;
+  transform: translate(-50%,-50%);
+  border:2px solid blueviolet;
+  color: black;
+  background-color: white;
+  box-shadow: 0px 0px 5px grey;
```

Comment:
Use variables for colors

Reply:
Yes but this is a component Color that why I have used direct color. Should I use color var for this?